src/components/wallet/earn/Validate/ValidatorInfo.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+    <div class="validator_card">
+        <Spinner v-if="loading" style="color: var(--primary-color)"></Spinner>
+
+        <div v-if="!loading" class="validator_child_card">
+            <div class="validator_info">
+                <div class="alt_validator_info">
+                    <div>
+                        <Tooltip style="display: inline-block" text="Staking Start Date">
```

Comment:
Let's use translation keys in `text` content, same for other `text` props in this component